packages/components/src/box-control/all-input-control.tsx

```diff
@@ -70,17 +94,34 @@ export default function AllInputControl( {
 	};
 
 	return (
-		<UnitControl
-			{ ...props }
-			disableUnits={ isMixed }
-			isOnly
-			value={ allValue }
-			onChange={ handleOnChange }
-			onUnitChange={ handleOnUnitChange }
-			onFocus={ handleOnFocus }
-			onHoverOn={ handleOnHoverOn }
-			onHoverOff={ handleOnHoverOff }
-			placeholder={ allPlaceholder }
-		/>
+		<HStack>
+			<UnitControl
+				{ ...props }
+				disableUnits={ isMixed }
+				id={ inputId }
+				isOnly
+				value={ allValue }
+				onChange={ unitControlOnChange }
```

Comment:
Not sure if this was intentional, but it looks like we have a separate `unitControlOnChange` function which is basically just a renaming of `onValueChange`. Maybe it was to balance it out with `sliderOnChange`, but perhaps not worth the extra overhead?

```suggestion
				onChange={ onValueChange }
```

Reply:
It was probably the easiest way to get around the TS signature of `UnitControlProps[ 'onChange' ]`

Reply:
Maybe it was an issue along the way? 🤔  But as of now `onChange={ onValueChange }` passes type checks.